src/layout/page/pageActionBar.jsx

```diff
@@ -46,7 +48,7 @@ function PageActionBar(props) {
     }, []);
 
     const toggleSmSearchVisibleClassName = (isVisible) => {
-        const containerClassName = `.${UI_CLASS_NAME}.${BEM_CONTAINER}`;
+        const containerClassName = `.ui.${BEM_CONTAINER}`;
```

Comment:
Deliberately hard-coding here and not using the constant?

Reply:
Yes, this is right, because `UI_CLASS_NAME` value changed to `cmui`.

Reply:
I may need to look into other components that are using this global const.

Reply:
Did some digging and it we are all good. This hard coded class name is good and all the others as well that are hard coded and or using the new `UI_CLASS_NAME` const.

Reply:
I trust you.  Resolving this thread.